handbook/company/communications.md

```diff
@@ -1519,6 +1519,11 @@ Use dashes (at least 3) to separate the header, and add colons to align the text
 |:---|---:|:---:|
 | Left alignment | Right alignment | Center Alignment |
 
+> #### Tables in the Fleet REST API documentation
```

Comment:
I think we can either make this it's own section or remove the title and leave it as a call-out box